DependencyInjection/Configuration.php

```diff
@@ -787,4 +792,13 @@ private function getAutoGenerateModes(): array
             'values' => $valuesArray,
         ];
     }
+
+    private function getDefaultSchemaManagerFactory(): string
+    {
+        if (class_exists(LegacySchemaManagerFactory::class)) {
+            return 'doctrine.dbal.legacy_schema_manager_factory';
```

Comment:
is this what you had in mind @ostrolucky ?

Reply:
yes